services/replicator/inconn.go

```diff
@@ -150,9 +150,6 @@ func (conn *inConnection) writeMsgsStream() {
 				return
 			}
 		} else {
-			// Note: we check closeChannel only in the default case.
```

Comment:
this is not a concern any more since we don't close the channel now in replicator (store will close the channel when seal marker is read)